jobs/build_docker/get_docker_commit_version.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash 
+set -x
```

Comment:
Don' put "set -x" if no necessary.
it will cause the Jenkins console exposition and very hard to read.